src/main.rs

```diff
@@ -1,9 +1,23 @@
 extern crate openzwave_stateful as openzwave;
 use openzwave::InitOptions;
-use openzwave::{ ValueGenre, ValueID };
+use openzwave::{ ValueGenre, ValueID, ZWaveNotification };
 
-use std::io;
+use std::{ io, thread };
 use std::io::Write;
+use std::sync::mpsc;
+
+fn display_prompt() {
+    print!("> ");
+    io::stdout().flush().unwrap(); // https://github.com/rust-lang/rust/issues/23818
+}
+
+fn spawn_notification_thread(rx: mpsc::Receiver<ZWaveNotification>) {
+    thread::spawn(move || {
+        for notification in rx {
+            println!("{}", notification);
+        }
+    });
```

Comment:
This feels like a waste of a thread. If we were going to spawn a thread to process notifications, shouldn't it be done when the notification is received from the OZW library?